tests/admin-mode_test.go

```diff
@@ -176,7 +167,7 @@ var _ = Describe("BDD of operator reconcile resiliency check", func() {
 
 			//Checking the chaosresult
 			By("Checking the chaosresult")
-			chaosResult, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
+			chaosResult, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+"pod-delete", metav1.GetOptions{})
```

Comment:
here we should have experimentName, as the c_result=engine_name+ exp_name and now name of Exp has been changed?

Reply:
This is due to this: https://github.com/litmuschaos/litmus/blob/master/experiments/generic/pod_delete/pod_delete_ansible_logic.yml#L6